### PR TITLE
Implemented dissalowing sessions in WCPM and fixed back button on installer in the sso configuration

### PR DESCRIPTION
**Pull request by bakejeyner**

### CODE REVIEW

File: ui/packages/ui/src/main/webapp/components/installer/installer.view.js [hunk 1]
```diff
@@ -92,9 +92,9 @@ define([
     regions: {
```
Comment (bakejeyner) [inline ui/packages/ui/src/main/webapp/components/installer/installer.view.js:92]: 📘 These are just re-ordering changes.

File: distribution/docs/src/main/resources/content/_managing/_configuring/web-context-policy-manager.adoc [hunk 2]
```diff
@@ -23,7 +23,7 @@ The guest principal will be used either by itself or along with any other princi
 
 === Session Storage
 Enabling session storage allows Jetty to enable the JSESSIONID cookie to maintain the user login.
```
Comment (blen-desta) [inline distribution/docs/src/main/resources/content/_managing/_configuring/web-context-policy-manager.adoc]: ✏️ This part isn't necessarily true anymore right. JSESSIONID stuff is always enabled we just don't look at it anymore

File: distribution/docs/src/main/resources/content/_managing/_configuring/web-context-policy-manager.adoc [hunk 3]
```diff
@@ -23,7 +23,7 @@ The guest principal will be used either by itself or along with any other princi
...
-
+Note that the `SAML` and `OIDC` authorization types require session storage to be enabled.
```
Comment (blen-desta) [inline distribution/docs/src/main/resources/content/_managing/_configuring/web-context-policy-manager.adoc]: ✏️ authentication not authorization

File: distribution/docs/src/main/resources/content/_reference/_tables/PolicyManager.adoc [hunk 4]
```diff
@@ -33,7 +33,7 @@
 |Allow Session Storage
 |sessionAccess
 |Boolean
-|Allow for session cookies to be used.
+|Allow for session cookies to be used. Note that the SAML and OIDC authorization types require session storage to be enabled.
```
Comment (blen-desta) [inline distribution/docs/src/main/resources/content/_reference/_tables/PolicyManager.adoc]: ✏️ authentication not authorization

File: platform/security/policy/security-policy-context/src/main/resources/OSGI-INF/metatype/metatype.xml [hunk 5]
```diff
@@ -29,7 +29,7 @@
             default="true"/>
 
         <AD id="sessionAccess" type="Boolean" name="Allow Session Storage"
-            description="Allow for session cookies to be used."
+            description="Allow for session cookies to be used. Note that the SAML and OIDC authorization types require session storage to be enabled."
```
Comment (blen-desta) [inline platform/security/policy/security-policy-context/src/main/resources/OSGI-INF/metatype/metatype.xml]: ✏️ authentication not authorization

**Review by stustison — APPROVED** [review]

**Review by blen-desta — APPROVED** [review]

**Pull request merged**